cli/src/action/api/rbac.rs

```diff
@@ -413,6 +413,37 @@ pub fn update_role(base_url: &str, auth: &str, role_update: RoleUpdate) -> Resul
         })
 }
 
+pub fn delete_role(base_url: &str, auth: &str, role_id: &str) -> Result<(), CliError> {
+    Client::new()
+        .delete(&format!("{}/authorization/roles/{}", base_url, role_id))
+        .header("SplinterProtocolVersion", RBAC_PROTOCOL_VERSION)
+        .header("Authorization", auth)
+        .send()
+        .map_err(|err| CliError::ActionError(format!("Failed to delete role {}: {}", role_id, err)))
+        .and_then(|res| {
+            let status = res.status();
+            if status.is_success() {
+                Ok(())
+            } else {
+                let message = res
+                    .json::<super::ServerError>()
+                    .map_err(|_| {
+                        CliError::ActionError(format!(
+                            "Get role delete request failed with status code '{}', but error \
```

Comment:
This error message looks a little wonky - seems like a `GET` request was just made

Reply:
Fixed